nautobot/extras/tests/test_views.py

```diff
@@ -925,28 +956,45 @@ def setUpTestData(cls):
         ScheduledJob.objects.create(
             name="test1",
             task="nautobot.extras.jobs.scheduled_job_handler",
-            job_class="-",
+            job_class="local/test_views/TestJob",
             interval=JobExecutionType.TYPE_IMMEDIATELY,
             user=user,
             start_time=datetime.now(),
         )
         ScheduledJob.objects.create(
             name="test2",
             task="nautobot.extras.jobs.scheduled_job_handler",
-            job_class="-",
+            job_class="local/test_views/TestJob",
             interval=JobExecutionType.TYPE_IMMEDIATELY,
             user=user,
             start_time=datetime.now(),
         )
         ScheduledJob.objects.create(
             name="test3",
             task="nautobot.extras.jobs.scheduled_job_handler",
-            job_class="-",
+            job_class="local/test_views/TestJob",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            start_time=datetime.now(),
+        )
+        # this should not appear, since it’s not enabled
+        ScheduledJob.objects.create(
+            enabled=False,
+            name="test4",
+            task="nautobot.extras.jobs.scheduled_job_handler",
+            job_class="local/test_views/TestJob",
             interval=JobExecutionType.TYPE_IMMEDIATELY,
             user=user,
             start_time=datetime.now(),
         )
 
+    def test_only_enabled_is_listed(self):
+        self.add_permissions("extras.view_scheduledjob")
+
+        response = self.client.get(self._get_url("list"))
+        self.assertHttpStatus(response, 200)
+        self.assertNotIn("test4", extract_page_body(response.content.decode(response.charset)))
```

Comment:
Should we also assert that test1 through test3 *are* in the page body?

Reply:
AFAIK this is already tested by the base class, right?

Reply:
True, to a limited extent - `test_list_objects_with_permission` doesn't, but `test_list_objects_with_constrained_permission` does. Good enough I guess.